docs/licenses.md

```diff
@@ -0,0 +1,8 @@
+### Dependences compatibility by license
+
+| Name | License | Compatibility |
+|------|---------|---------------|
+| OpenBlas | BDS-3 | Yes |
+| Blaze | BSD-3 | Yes |
+| libjpeg-turbo | [It's complicated](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/LICENSE.md)| Yes(?) |
```

Comment:
Don't see any problem with libjepeg: 
> The IJG (Independent JPEG Group) License, which is listed in [README.ijg](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/README.ijg)
This license applies to the libjpeg API library and associated programs (any code inherited from libjpeg, and any modifications to that code.)

We don't inherit the code base and don't distribute with the wavelet buffer. I'm pretty sure we compatible.

Reply:
There are three different licenses for jpeg api, jpeg-turbo api and SIMD instructions.
All of them are pemissive and we could use them. But maybe we must mention them.

Reply:
yes, count them and and add links